src/fusor/examples/bcr_abl1.json

```diff
@@ -1,111 +1,131 @@
 {
   "type": "CategoricalFusion",
-  "structural_elements": [
-    {
-      "type": "TranscriptSegmentElement",
-      "transcript": "refseq:NM_004327.3",
-      "gene_descriptor": {
-        "type": "GeneDescriptor",
-        "id": "normalize.gene:BCR",
-        "gene_id": "hgnc:1014",
-        "label": "BCR"
+  "structure": {
+    "type": "Adjacency",
+    "adjoinedSequences": [{
+      "type": "SequenceLocation",
+      "sequenceReference": {
+        "id": "GRCh38:chr22",
+        "type": "SequenceReference",
+        "refgetAccession": "SQ.7B7SHsmchAR0dFcDCuSFjJAo7tX87krQ",
+        "residueAlphabet": "na"
       },
-      "element_genomic_end": {
-        "id": "fusor.location_descriptor:NC_000022.11",
-        "type": "LocationDescriptor",
-        "label": "NC_000022.11",
-        "location": {
-          "type": "SequenceLocation",
-          "sequence_id": "refseq:NC_000022.11",
-          "interval": {
-            "type": "SequenceInterval",
-            "start": {
-              "type": "Number",
-              "value": 23253980
-            },
-            "end": {
-              "type": "Number",
-              "value": 23253981
+      "end": 23290413,
```

Comment:
Note that here we use `end` (appropriately) but the 5' partner expressed in genomic coordinates should use `start` if the transcribed sequence is on the negative strand. Using `end` represents a boundary that includes sequence to the left (here, the sequence left of the aligned genomic coordinates at position 23290413, corresponding to the BCR exon 14). 

<img width="788" alt="image" src="https://github.com/user-attachments/assets/3810882b-7d6c-495f-8dc7-2e39fe6e5bb9">